lesson_1.1/src/App.js

```diff
@@ -1,41 +1,48 @@
-import React,{useState, useEffect} from 'react'
```

Comment:
I'd still keep this named imports from react since you're using useEffect more than once and useState more than once.